pom.xml

```diff
@@ -442,6 +443,90 @@ of Jackson components maintained by FasterXML.com
     </dependencies>
   </dependencyManagement>
 
+  <profiles>
+    <profile>
+      <id>windows</id>
+      <activation>
+        <os>
+          <family>Windows</family>
+        </os>
+      </activation>
+      <properties>
+        <gradle.wrapper>gradle.bat</gradle.wrapper>
+      </properties>
+    </profile>
+    <profile>
+      <id>posix</id>
+      <activation>
+        <os>
+          <family>unix</family>
+        </os>
+      </activation>
+      <properties>
+        <gradle.wrapper>./gradlew</gradle.wrapper>
+      </properties>
+    </profile>
+    <profile>
+      <id>jackson-gradle-catalog-and-platform</id>
+      <activation>
+        <file>
+          <exists>${basedir}/gradle</exists>
+        </file>
+      </activation>
+      <build>
+        <plugins>
```

Comment:
Would be nice to have brief Javadoc referencing this PR, explaining what is being done.